src/iiif/views.py

```diff
@@ -45,11 +47,17 @@ def send_dataportaal_login_url_to_mail(request):
         token = authentication.create_mail_login_token(email, settings.JWT_SECRET_KEY)
         login_url = origin_url + '?auth=' + token
 
-        # Send the email
-        email_subject = "Toegang bouw- en omgevingsdossiers data.amsterdam.nl"
-        email_body = render_to_string('login_link.html', {'login_url': login_url})
-        # TODO: move actually sending the email to a separate process
-        mailing.send_email(payload['email'], email_subject, email_body)
+        # Store a job to send the email
+        email_info = {
+            'email': payload['email'],
+            'subject': "Toegang bouw- en omgevingsdossiers data.amsterdam.nl",
+            'body': render_to_string('login_link.html', {'login_url': login_url}),
+        }
+        collection = Collection.objects.get(name=settings.EMAIL_COLLECTION_NAME)
```

Comment:
Is het wenselijk hier `get_or_create()` te gebruiken?

Reply:
Dat kan idd ook. Heb het aangepast.

Reply:
Ik zou zeggen van niet. Ik wil het creëren van collections wel in de hand houden en niet vanuit de code doen. Bovendien zou je dan een collection aanmaken die defaults heeft waar die wellicht anders zou willen.